lumeer-storage/lumeer-storage-mongodb/src/main/java/io/lumeer/storage/mongodb/model/MorphiaDocument.java

```diff
@@ -198,4 +201,21 @@ public String toString() {
             ", data=" + data +
             '}';
    }
+
+   public org.bson.Document toBsonDocument() {
+      DataDocument dataDocument = new DataDocument(COLLECTION_ID, collectionId)
+            .append(CREATED_BY, createdBy)
+            .append(CREATION_DATE, convertLocalDateTimeToDate(creationDate))
+            .append(UPDATED_BY, updatedBy)
+            .append(UPDATE_DATE, convertLocalDateTimeToDate(updateDate))
+            .append(DATA_VERSION, dataVersion);
+      return MongoUtils.dataDocumentToDocument(dataDocument);
```

Comment:
You are in Mongo module here so you can directly create `org.bson.Document` and do not waste resources by converting it. :)